sdk/eventhub/event-hubs/test/utils/constants.ts

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+export enum EnvVarKeys {
+  EVENTHUB_FQDN = "EVENTHUB_FQDN",
+  EVENTHUB_NAME = "EVENTHUB_NAME",
+  EVENTHUB_CONNECTION_STRING = "EVENTHUB_CONNECTION_STRING",
+  TEST_MODE = "TEST_MODE",
+}
+
+// Mocked values
```

Comment:
nitpick: could we group these values into a `mocked` object to make it clear that they are not real values?

Reply:
Good point, they're imported once so I made it a named import in https://github.com/Azure/azure-sdk-for-js/pull/30421/commits/1a3b76134a7fc6f69a7f1e078110622388795b60.